foods = ('muffin','eggs','sausage','coffee','bagels')
for food in foods:
    print(food)

#food[2]=pancakes (This is the error)

new_foods = ('muffin','eggs','bacon','coffee','fruit')

print("\nnew_foods:")
for food in new_foods:
    print(food)